components/textfield/stories/template.js

```diff
@@ -60,7 +69,18 @@ export const Template = ({
       ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
     })}
       style=${ifDefined(styleMap(styles))}
-      @click=${onclick}>
+      @click=${onclick}
+      @focusin=${(e) => {
+        const textfield = e.target.closest(`.${rootClass}`);
```

Comment:
Can we create a single function for focusin and focusout? We are repeating most of the code!!!